include/xrpl/protocol/detail/features.macro

```diff
@@ -29,6 +29,8 @@
 // If you add an amendment here, then do not forget to increment `numFeatures`
 // in include/xrpl/protocol/Feature.h.
 
+// Check flags in Credential transactions
+XRPL_FIX    (CredentialFlags,            Supported::yes, VoteBehavior::DefaultYes)
```

Comment:
All amendments should be `DefaultNo` unless it's an emergency.

Reply:
@mvadari please check it with @Bronek

Reply:
General practice is that we only use `DefaultYes` if the bug is of great severity and we want everyone to update ASAP. That is not true of this PR.

Reply:
I do not feel strongly about `DefaultYes` - looked at the historical pattern of fixes (as opposed to new features). The flags provided in the transaction will not be persisted, so the potential damage caused by users issuing transactions with a flags which are next ignored is transient only. Nothing gets corrupted, so it is not urgent that we fix it.

I'd be happy to re-approve with `DefaultNo`

Reply:
fixed